components/lock.rst

```diff
@@ -204,12 +203,12 @@ Automatically Releasing The Lock
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Locks are automatically released when their Lock objects are destroyed. This is
-an implementation detail that will be important when sharing Locks between
+an implementation detail that is important when sharing Locks between
 processes. In the example below, ``pcntl_fork()`` creates two processes and the
 Lock will be released automatically as soon as one process finishes::
 
     // ...
-    $lock = $factory->createLock('report-generation', 3600);
+    $lock = $factory->createLock('pdf-creation', 3600);
```

Comment:
```suggestion
    $lock = $factory->createLock('pdf-creation', ttl: 3600);
```

Reply:
My goal was not to add names everywhere but only when we introduce the argument or when it makes sense to better understand the code. Here, I will remove the argument instead as it does not add anything interesting in the example.